conda/meta.yaml

```diff
@@ -17,16 +17,16 @@ requirements:
   host:
     - pip
     - pytest-runner
-    - python >=3.6,<3.9
+    - python >=3.6,<3.10
     - numpy >=1.18.0,<2
-    - pandas >=1.1,<1.1.5
-    - pytorch >=1.4,<2
+    - pandas >=1.1.3,<1.1.5
+    - pytorch >=1.7.1,<2
```

Comment:
In the spreadsheet we have the minimum as `1.4`. Why did we need it to be `1.7.1` and should it be updated for all the libraries?

Reply:
Yes, I believe it does: https://download.pytorch.org/whl/torch_stable.html